src/js/vic-v2/components/CropperController.jsx

```diff
@@ -131,13 +137,19 @@ export default class CropperControls extends React.Component {
     };
   }
 
-  componentWillUpdate(nextProps) {
-    if (nextProps.windowWidth !== this.props.windowWidth) {
-      const cropper = this.refs.cropper;
-      if (cropper) {
-        this.setCropBox();
-      }
-    }
+  componentWillMount() {
+    this.detectWidth();
+  }
+
+  componentDidMount() {
+    // updating the width state triggers Cropper remount- debounce event to reduce jank during active resize
+    this.debouncedDetectWidth = this.debouncedDetectWidth || _.debounce(this.detectWidth, 250);
+    window.addEventListener('resize', this.debouncedDetectWidth);
```

Comment:
Couldn't completely get away from detecting the windowWidth but now it's _only_ used to manage remounting the Cropper- it doesn't support changing the width of the canvas once its mounted.

Cropper uses `windowWidth` as a key to force the component to remount and redraw the canvas when the width changes. The only use case for the window width changing is orientation change on a mobile device or if someone is actively dragging a desktop browser. This debounces the event so that typically happens just once- after the drag or orientation change is complete. 

Tried a few timings- anything less than 200ms was too bouncy.